CHANGELOG.rst

```diff
@@ -5,6 +5,10 @@ History
 Unreleased
 ==========
 
+* Add `Travis CI integration`_ and fix related test issues.
```

Comment:
This is fine, as we know this is going in the next release, though generally we shouldn't update the changelog as we don't know when a PR will be merged.

Reply:
Yeah, ideally reno or towncrier should be configured to take care of this :)

Reply:
One step at a time :)